### PR TITLE
✨ (up-integration): Added initial stub

### PR DESCRIPTION
**Pull request by Grobbies26**
Added stub for the get Academic record for future years use once approval from varsity has been acquired. Please have a look @COS-301/c-1

### CODE REVIEW

**Comment by Grobbies26** [comment]
Please advise on the second commit (merge branch...) if that will cause an issue with the repo, I believe that it is due to the fact that a commit occurred while I was halfway through the work I was doing and hence was unable to pull the new data into my local copy without commiting what I already had. If it will cause an issue I will refork the main repo and just redo the work quickly.

**Comment by arneschreuder** [comment]
But I love it. There are labels, the title is almost correct. I like it.

**Comment by arneschreuder** [comment]
@Grobbies26 Also, make sure to just check that your feature branch is up to date with dev and if not, pull and push.

**Review by u20437863-James-Butler — DISMISSED** [review]
Sorry for the delay. Well done on completing this so quickly! I can't find any issues beyond what @arneschreuder has mentioned already

File: libs/api/upintegration/service/feature/src/lib/api-upintegration-service-feature.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@nestjs/common';
```
[the remark this answers is not in the document]
Comment (Grobbies26) [reply libs/api/upintegration/service/feature/src/lib/api-upintegration-service-feature.ts:1]: How would one export the module and the service in this one file, or am I misunderstanding

**Comment by arneschreuder** [comment]
Please update branch. PR is getting stale.

**Comment by Grobbies26** [comment]
Could you provide clarity on why it failed the first test, i cant make sense of what that means

**Comment by arneschreuder** [comment]
Please run for me: https://nx.dev/cli/format-check to check?

**Comment by arneschreuder** [comment]
Have you run:

* git pull upstream develop
* git push origin feature/....?

**Comment by Grobbies26** [comment]
`● Validation Error:

  Module <rootDir>/src/test-setup.ts in the setupFilesAfterEnv option was not found.
         <rootDir> is: /home/runner/work/graduates/graduates/libs/api/upintegration/api/shared/data-access

  Configuration Documentation:
  https://jestjs.io/docs/configuration`

This is the error its giving me, Not sure how to interpret

**Comment by Grobbies26** [comment]
I believe I have corrected all outstanding issues

File: libs/api/upintegration/api/feature/src/lib/api-upintegration.resolver.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Query, Resolver } from "@nestjs/graphql";
+import { ApiUpIntegrationServiceFeatureModule} from '../../../../../../api/upintegration/service/feature/src/lib/api-upintegration-service-feature'
+import{ApiUpIntegration} from '../../../../api/shared/data-access/src/lib/api-upintegration.entity'
+@Resolver(of => ApiUpIntegration)
+export class ApiUpIntegrationResolver {
+   constructor(private upintegrationService:ApiUpIntegrationServiceFeatureModule ){}
```
[the remark this answers is not in the document]
Comment (arneschreuder) [reply libs/api/upintegration/api/feature/src/lib/api-upintegration.resolver.ts:7]: This is still not right, but I will merge this, you can fix this later.

**Review by arneschreuder — APPROVED** [review]

**Comment by arneschreuder** [comment]
@Grobbies26 go ahead and merge.

**Review by u20437863-James-Butler — APPROVED** [review]
Well done on fixing Arné's queries and getting passed all of the tests

**Pull request merged**